biojava-structure/src/main/java/org/biojava/nbio/structure/basepairs/BasePairParameters.java

```diff
@@ -235,6 +235,127 @@ public String getPairSequence() {
     }
 
     /**
+     * Return the buckle in degrees for the given base pair
+     * @param bp the number of the base pair (starting with 0)
+     * @return the value as a double (in degrees)
+     */
+    public Double getBuckle(int bp) {
+        if (bp < 0 || bp >= getPairingParameters().length) return null;
```

Comment:
Better throw an exception (`IllegalArgumentException` or `IndexOutOfBoundsException`, as you prefer) with a message describing what happened. The same for all other getters.